cog.c

```diff
@@ -167,7 +170,9 @@ on_handle_local_options (GApplication *application,
     }
 
     const char *uri = NULL;
-    if (!s_options.arguments) {
+    if (s_options.automation) {
+        uri = "about:blank";
```

Comment:
Could we just check whether CogShell object is automated? This would avoid the need for the option list entry.

Reply:
Checking the CogShell for automation is also possible. I've updated the patch with it.

Regarding the option list entry, I thought about it more like a way to document this flag, keeping consistent with other arguments (even though it has this special check before the application is created).